src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculation {
```

Comment:
if we implement AreaCalculator in Figure class we will not need to implement it to every separate figure class because they will inherit it